src/evaluation/WorkspaceManager.jl

```diff
@@ -22,9 +22,9 @@ Base.@kwdef mutable struct Workspace
 end
 
 "These expressions get evaluated whenever a new `Workspace` process is created."
-const process_preamble = quote
+process_preamble() = quote
```

Comment:
This needed to change to a function such that the path could be queried at runtime rather than at `using`-time which would pick the wrong path to `include`.

Reply:
Actually, I just was planning to send in a PR to change this to a function too because it now allows users to override the preamble :rofl: What a coincidence. Thanks!